ivadomed/transforms.py

```diff
@@ -12,6 +11,7 @@
 from scipy.ndimage.measurements import label, center_of_mass
 from scipy.ndimage.morphology import binary_dilation, binary_fill_holes, binary_closing
 from skimage.exposure import equalize_adapthist
+from skimage import measure
 from torchvision import transforms as torchvision_transforms
 
```

Comment:
Currently, your branch is conflicting with the `master` branch. 

![Screenshot from 2021-07-02 11-08-22](https://user-images.githubusercontent.com/16181459/124294588-dd6b4600-db25-11eb-9d08-d67fc32f9f05.png)

Could you please merge the changes from the branch `ivadomed:master` into your branch, and resolve the conflicts?

Reply:
Also, in the future, could you please check for these conflicts and fix them when you see them?

Ideally, the PR should never have conflicts, so when you see this message on your PR, it is helpful to try fix it immediately (so that the people reviewing your PR won't have to take the time to ask).